docs/bitcoin/manage.md

```diff
@@ -0,0 +1,33 @@
+# Managing Bitcoin
+
+Bitcoin runs automatically when MyNode starts. By clicking on the "Manage" button under Bitcoin on the home page, you can view the current state of Bitcoin daemon and interact with it. You can also reach the page by pointing to **\<MYNODE-IP\>/bitcoind**
```

Comment:
Bitcoin runs automatically when myNode starts. By clicking on the "Manage" button under Bitcoin on the home page, you can view the current state of the Bitcoin blockchain and your node, and interact with your node. You can also reach the page by pointing to **\<MYNODE-IP\>/bitcoind**

Reply:
added